src/pomodoro.sh

```diff
@@ -85,10 +85,13 @@ function show_active_pomodoro_timebox()
 # Show registered tags with number identification.
```

Comment:
Think you forgot to add verbose to `select_from` in the function `show_active_pomodoro_timebox` that `kw pomodoro --check-timer` uses underneath.

It is
![Screenshot from 2024-01-17 14-06-43](https://github.com/kworkflow/kworkflow/assets/95055784/e5ffc725-19f1-4231-b964-46d3e05532fe)


but it should be
![Screenshot from 2024-01-17 14-09-03](https://github.com/kworkflow/kworkflow/assets/95055784/76e39943-80d0-46ab-b2b6-896b02505cdf)